features/cli/display_failing_result.feature

```diff
@@ -6,5 +6,18 @@ Feature: Display failing result
     Then it should fail with:
       """
       ✗ http://localhost:54321/missing_header.html
-        - Found 0 h1 elements.
+        * Headings > Exactly one main heading
+          - Found 0 h1 elements.
+      """
+
+  Scenario: Two standards fail
+    Given a website running at http://localhost:54321
+    When I run `a11y http://localhost:54321/two_headings_failures.html`
+    Then it should fail with:
+      """
+      ✗ http://localhost:54321/two_headings_failures.html
+        * Headings > Content must follow headings
```

Comment:
Yeah I like the whitespace too.
On Fri, 26 Feb 2016 at 14:20, Josh Chisholm <notifications@github.com>
wrote:

> In features/cli/display_failing_result.feature
> <https://github.com/cucumber-ltd/bbc-a11y/pull/107#discussion_r54248212>:
>
> > @@ -6,5 +6,18 @@ Feature: Display failing result
> >      Then it should fail with:
> >        """
> >        ✗ http://localhost:54321/missing_header.html
> > -        - Found 0 h1 elements.
> > +        * Headings > Exactly one main heading
> > +          - Found 0 h1 elements.
> > +      """
> > +
> > +  Scenario: Two standards fail
> > +    Given a website running at http://localhost:54321
> > +    When I run `a11y http://localhost:54321/two_headings_failures.html` <http://localhost:54321/two_headings_failures.html>
> > +    Then it should fail with:
> > +      """
> > +      ✗ http://localhost:54321/two_headings_failures.html
> > +        * Headings > Content must follow headings
>
> Sure, colons.
>
> Most other a11y tools don't have any whitespace in the name of the
> standard/sniff, it's more of an identifier, presumably so they can be
> ignored without any confusion over whitespace. Personally I quite like the
> humanised names, instead of something like
> Headings.ContentMustFollowHeadinhgs. WDYT?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cucumber-ltd/bbc-a11y/pull/107/files#r54248212>.
>

-- 



The Cucumber logo is the intellectual property of Cucumber Ltd, a limited 
company registered in Scotland, number 456793.

UK Headquarters: Cucumber Ltd, Drumsyniebeg, Lochgoilhead, Cairndow, 
Argyll, PA24 8AN UK.

CONFIDENTIALITY NOTICE: The information in this e-mail is confidential and 
privileged; it is intended for use solely by the individual or entity named 
as the recipient hereof. Disclosure, copying, distribution, or use of the 
contents of this e-mail by persons other than the intended recipient is 
strictly prohibited and may violate applicable laws. If you have received 
this e-mail in error, please delete the original message and notify us by 
email immediately. Thank you. Cucumber Ltd.